src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -638,9 +633,54 @@ public void addRtpExtension(RtpExtension rtpExtension)
     }
 
     @Override
-    public void setVideoConstraints(Map<String, VideoConstraints> newVideoConstraints)
+    public void setSenderVideoConstraints(ImmutableMap<String, VideoConstraints> newVideoConstraints)
```

Comment:
I think the `sender/receiver` terminology makes things very confusing. After going through the code and sketching things on a piece of paper, I understand that:

`e.setSenderVideoConstraints()` sets the constraints that `e` wishes to receive from other senders. While `e.receiverVideoConstraintsChanged()` sets the constraints that endpoints receiving video from `e` wish to impose. I think that's consistent, but somewhat confusing and not documented.

I think using "local/remote" language may be clearer (along with docs that say the basics in a simple way). I.e. 
```
/*
Sets the constraints to be applied to the streams sent by this endpoint. Each entry in the map is a constraint imposed by a receiving endpoint.`
*/
setLocalVideoConstraints(Map<String, VideoConstraints>)

/**
Sets the constraints that this endpoint wishes to impose on streams sent by remote endpoints.
*/
setRemoveVideoConstraints()
```

If you don't like this language that's okay, it's just an example. But please add a simple description of the semantics in the docs.

Reply:
I think of it in terms of rtp sender/receiver, i.e. conceptually the setSenderVideoConstraints sets the video constraints on the rtp sender and the setReceiverVideoConstraints sets the video constraints on the rtp receiver (we no longer have such a method because it got simplified in the previous pr). Does that make more sense?

That said, I'll admit I'm not very good at naming, so I'm happy to follow through your suggestion if you insist.

Reply:
> I think of it in terms of rtp sender/receiver, i.e. conceptually the setSenderVideoConstraints sets the video constraints on the rtp sender and the setReceiverVideoConstraints sets the video constraints on the rtp receiver

This description makes it much easier to understand!